distmaker/dists/common.sh

```diff
@@ -211,7 +211,7 @@ function dm_generate_bower() {
 function dm_generate_vendor() {
   local repo="$1"
   pushd "$repo"
-    ${DM_COMPOSER:-composer} install
+    ${DM_COMPOSER:-composer} install --ignore-platform-reqs
```

Comment:
Agree `--ignore-platform-reqs` isn't really great. I've upgraded PHP on `test-ubu1204-5` to 5.3.27, rebased to remove `--ignore-platform-reqs`, and pushed.